repl/scala-2.11/src/main/scala/org/apache/spark/repl/SparkILoop.scala

```diff
@@ -97,13 +97,13 @@ class SparkILoop(in0: Option[BufferedReader], out: JPrintWriter)
   override def commands: List[LoopCommand] = sparkStandardCommands
 
   /**
-   * We override `loadFiles` because we need to initialize Spark *before* the REPL
+   * We override `createInterpreter` because we need to initialize Spark *before* the REPL
```

Comment:
Sorry, I meant you should follow the version changes. You can't take this change; it depends on 2.11.11